### ISSUE TITLE
Upload offical Python client to PyPI

### ISSUE THREAD

**Issue by abitrolly** [savoirfairelinux/opendht#500]
https://pypi.org/project/opendht/ version 1.8.1 is very old. Is it possible to upload new versions during release process?

@manuels can you help with that?

**Comment by abitrolly** [comment]
This this guide https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ seems pretty easy.

**Comment by manuels** [comment]
Maybe I can find the time in the next day. Currently v1.8.1 is published because it is default on debian.

**Comment by aberaud** [comment]
@manuels Thanks ! Version in Debian is obsolete and we're working to get it updated. A PyPI update would be great :-)

**Comment by ghenry** [comment]
Still a requirement?  

Thanks.

**Comment by abitrolly** [comment]
@ghenry yes, testing Python examples without building everything would be pretty much awesome. I am particularly interested in playing with local peer discovery https://github.com/savoirfairelinux/opendht/wiki/Peer-discovery

**Comment by manuels** [comment]
I'm sorry, I lost access to pypi. I have contacted them to regain access.
